public/index.html

```diff
@@ -17,59 +18,62 @@ <h1 class="header__wordmark">
           fluent
         </h1>
         <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          <a class="link" href="http://projectfluent.io/fluent/guide">Learn</a>
+          <a class="link" href="http://projectfluent.io/play">Play</a>
+          <a class="link" href="#implementation-docs">Use</a>
```

Comment:
I agree that "Gihub" and "Wiki" are not good links here, but I don't see how "Learn" and "Play" improve the UX. "Learn" what? The syntax? How to use it? More about Fluent? "Play" with what? Etc. The action verbs creates more questions in my mind. In comparison, "Syntax Guide" and "Online Editor" are pretty straight forward to me.